port/zos390/omrvmem.c

```diff
@@ -1389,7 +1409,8 @@ omrvmem_find_valid_page_size(struct OMRPortLibrary *portLibrary, uintptr_t mode,
 	}
 
 _end:
-	if (IS_VMEM_PAGE_FLAG_NOT_USED(*pageFlags)) {
+	if ((IS_VMEM_PAGE_FLAG_NOT_USED(*pageFlags)) ||
+		(0 != (OMRPORT_VMEM_PAGE_FLAG_PAGEABLE_PREFERABLE & (*pageFlags)))) {
```

Comment:
be careful here... `vmem_default_large_page_size_ex()` might clean up  `OMRPORT_VMEM_PAGE_FLAG_PAGEABLE_PREFERABLE` flag

Reply:
`default_large_page_size_ex()` will not clean up `OMRPORT_VMEM_PAGE_FLAG_PAGEABLE_PREFERABLE` since that flag is stored directly into `pageFlags`, while the result from `default_large_page_size_ex` is stored into `defaultLargePageFlags`, and is only used to compare `validflags` and `defaultLargePageFlags`.